01-Login/src/main/java/com/auth0/example/mvc/LogoutController.java

```diff
@@ -1,24 +1,48 @@
 package com.auth0.example.mvc;
 
+import com.auth0.example.security.AppConfig;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
 
 @SuppressWarnings("unused")
 @Controller
-public class LogoutController {
+public class LogoutController implements LogoutSuccessHandler {
+
+    @Autowired
+    private AppConfig appConfig;
 
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    @RequestMapping(value = "/logout", method = RequestMethod.GET)
```

Comment:
you no longer need to specify the `/logout` path?

Reply:
The `/logout` path is no longer needed as is used by default when logout is set [here](https://github.com/auth0-samples/auth0-spring-security-mvc-sample/pull/28/files#diff-282e204682cb8b240fa4d6ed758703d6R72).